dashboard/app/models/pd/enrollment.rb

```diff
@@ -120,7 +120,11 @@ def self.for_school_district(school_district)
   # Except for FiT workshops - no exit surveys for them!
   # This scope is used in ProfessionalLearningLandingController to direct the teacher
   #   to their latest pending survey.
-  scope :with_surveys, -> {for_ended_workshops.attended.where.not(pd_workshops: {subject: SUBJECT_FIT})}
+  scope :with_surveys, (lambda do
```

Comment:
Just curious: is the switch to `lambda` purely to get multi-line code support?

Reply:
Yeah; [Rubocop prefers](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/Lambda) curlies for one-lines, and `do ... end` for multiline, and it turns out lambda syntax is tied to one or the other.